core/src/main/scala/scalacss/Attrs.scala

```diff
@@ -2611,8 +2614,17 @@ object Attrs {
    *
    * @see <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/flex">MDN</a>
    */
-  final val flex = Attr.alias("flex", Transform keys CanIUse.flexbox)(_(
-    flexGrow, flexShrink, flexBasis))
+  object flex extends TypedAttrBase {
+    override val attr = Attr.alias("flex", Transform keys CanIUse.flexbox)(_(flexGrow, flexShrink, flexBasis))
+
+    type FlexBasis = ValueT[LenPctAuto]
+    val none: AV = av("none")
+    def apply(flexGrow: Int): AV = av(s"$flexGrow")
+    def apply(flexBasis: FlexBasis): AV = av(s"$flexBasis")
+    def apply(flexGrow: Int, flexBasis: FlexBasis): AV = av(s"$flexGrow $flexBasis")
+    def apply(flexGrow: Int, flexShrink: Int): AV = av(s"$flexGrow $flexShrink")
+    def apply(flexGrow: Int, flexShrink: Int, flexBasis: FlexBasis): AV = av(s"$flexGrow $flexShrink $flexBasis")
```

Comment:
`flexGrow` and `flexShrink` both extend `TypedAttrT1[Number]` which means instead of `Int` you should use `ValueT[Number]` (which accepts `double` and `int`).